# left image LDC LUT definition
left_lut_file_path:                 $(find ti_sde)/../ldc_lut/SN5867575_HD_LUT_left.bin

# right image LDC LUT definition
right_lut_file_path:                $(find ti_sde)/../ldc_lut/SN5867575_HD_LUT_right.bin

# input_format
# 0: U8 (bmp), 1: YUV422
input_format:                       1

# input data width in pixels
width:                              1280

# input data height in lines
height:                             720

# SDE type
# 0: Single-layer SDE, 1: Multi-layer SDE
sde_algo_type:                      0

# For multi-layer SDE only
# number of layers (2, 3)
num_layers:                         2

# For single-layer SDE only
# disparity threshold
sde_confidence_threshold:           1

####### SDE params BEGIN #######

# minimum disparity in pixels
# minimum disparity (0 -> 0 pixels, 1 -> -3 pixels)
disparity_min:                      0

# <disparity in pixels>, maximum disparity in pixels
# maximum disparity (0 -> 63 pixels, 1 -> 127 pixels, 2 -> 191 pixels)
disparity_max:                      1

# <0 - 255>, left-right consistency check threshold in pixels
threshold_left_right:               3

# <0 or 1>, disable/enable texture based filtering
texture_filter_enable:              0

# <0 - 255>, scaled threshold for texture based filtering
threshold_texture:                  0

# <0 - 127>, Optimization penalty constant for small disparity change
aggregation_penalty_p1:             32

# <0 - 255>, Optimization penalty constant for large disparity change
aggregation_penalty_p2:             192

# <0, 1>, median filter in SDE
median_filter_enable:               1

####### SDE params END #######

# <1 - 4>, pipeline depth
# single-layer SDE - 2
# multi-layer SDE  - 3
pipeline_depth:                     2

# interactive mode
is_interactive:                     0

# Flag to indicate if the graph should be exported
#  0 - disable
#  1 - enable
exportGraph:                        0

# Real-time logging enable.
#  0 - disable
#  1 - enable
rtLogEnable:                        0

# Core deployment for disparity merge node, when sde_algo_type = 1
disp_merge_deploy_core:             TIVX_TARGET_A72_0

# Core deployment for hole filling node, when sde_algo_type = 1
hole_filling_deploy_core:           TIVX_TARGET_A72_0






